shell/src/dapps/o-onboarding/processes/loginWithTorus.ts

```diff
@@ -132,11 +133,11 @@ const processDefinition = (processId: string) =>
             onDone: [
               {
                 cond: (context) => Environment.useMockLogin === true,
-                target: "useMockProfile",
+                target: "#useMockProfile",
```

Comment:
state name or state id?

Reply:
id. it has both, but we're using id everywhere else.